README.md

```diff
@@ -1,5 +1,124 @@
-# Build
-Extract the entire disc to `orig/G8ME01` using Dolphin.<br/>
-Run `./configure` to split and create the files necessary for building<br/>
-Then run `ninja` to build the dol<br/>
-To clear all build directories, `make` or `make clean` will delete all generated files<br/>
+Paper Mario: The Thousand-Year Door [![Discord Badge]][discord]
+=============
+
+<!--
+Replace with your repository's URL.
+-->
+[Build Status]: https://github.com/zeldaret/tww/actions/workflows/build.yml/badge.svg
+[actions]: https://github.com/zeldaret/tww/actions/workflows/build.yml
```

Comment:
Do we want to update the links to these?

Reply:
I haven't played with the GitHub CI and we don't have any shield access to upload decomp progress yet, so I kinda just left the metadata but removed the badges at the top
Can always ask in the server under #frogress to get those working and CI just involves having a private repo with game data so that it can actually build